### DISCUSSION TITLE
ax6的nss emc没有开关

### DISCUSSION THREAD

**Discussion by he110777** [Boos4721/openwrt#124] [Bugs/问题反馈]
如题。

**Comment by Boos4721** [comment]


**Reply by he110777** [reply]
流量分载显示未运行。全锥型 NAT，bbr可以正常开启。我在检验一下配置，或者编译ax3600的试试。

**Reply by he110777** [reply]
在app已经选上CONFIG_PACKAGE_luci-app-turboacc_INCLUDE_Shortcut_FE_for_ECM ，但编译好固件还是显示流量分载未运行，ax6和ax3600都是一样。
在Kernal - Network support的相关NSS选项是不是也要设置？

**Reply by Boos4721** [reply]
> 在app已经选上CONFIG_PACKAGE_luci-app-turboacc_INCLUDE_Shortcut_FE_for_ECM ，但编译好固件还是显示流量分载未运行，ax6和ax3600都是一样。
> 在Kernal - Network support的相关NSS选项是不是也要设置？

ofc

**Reply by he110777** [reply]
谢谢，我再试试

**Reply by Gaojianli** [reply]
ax3600手动勾上`qca-nss-ecm`内核模块后无法开机，会在开机的过程中panic

**Reply by he110777** [reply]
truboacc不要选，Shortcut_FE_for_ECM ，默认Shortcut_FE，驱动已经是nss的。

**Reply by Gaojianli** [reply]
> truboacc不要选，Shortcut_FE_for_ECM ，默认Shortcut_FE，驱动已经是nss的。

<img width="570" alt="image" src="https://user-images.githubusercontent.com/19235246/168457979-3bcd7f92-51e1-4d16-be25-1bd6c12d5944.png">
不好意思不太懂您的意思，能详细说下怎么勾选吗？ @he110777

**Reply by Deleted-Account** [reply]
是不是NSS没有了？我前两天按上游默认设置编译的，没有动设置。编译好后不保存配置刷到AX6后进去看TurboACC是SFE，有开关，不是NSS，测试局域网千兆有线传输，CPU占用22%这样，不像NSS那样不占用CPU

**Reply by MaxMaoSu** [reply]
> 选了Shortcut_FE_for_ECM后需要手动勾选kmod-shortcut-fe-drv

朋友，请问现在勾选kmod-shortcut-fe-drv后AX6刷入正常了吗？之前编译勾选后刷入会无限重启
